trieste/models/keras/sampler.py

```diff
@@ -114,7 +114,8 @@ class deep_ensemble_trajectory(TrajectoryFunctionClass):
     Option `diversify` can be used to increase the diversity in case of optimizing very large
     batches of trajectories. We use quantiles from the approximate Gaussian distribution of
     the ensemble as trajectories, with randomly chosen quantiles approximating a trajectory and
-    using a reparametrisation trick to speed up computation. Note that quantiles are not true
+    using a reparametrisation trick to speed up computation. Only epistemic uncertainty is taken
+    into account in sampling. Note that quantiles are not true trajectories, so this will likely
     trajectories, so this will likely have some performance costs.
```

Comment:
```suggestion
    using a reparametrisation trick to speed up computation. Only epistemic uncertainty is taken
    into account in sampling. Note that quantiles are not true trajectories, so this will likely
    have some performance costs.
```